aws_lambda_powertools/utilities/jmespath_utils/envelopes.py

```diff
@@ -6,3 +6,8 @@
 CLOUDWATCH_EVENTS_SCHEDULED = EVENTBRIDGE
 KINESIS_DATA_STREAM = "Records[*].kinesis.powertools_json(powertools_base64(data))"
 CLOUDWATCH_LOGS = "awslogs.powertools_base64_gzip(data) | powertools_json(@).logEvents[*]"
+S3_SNS_SQS = "Records[*].powertools_json(body).powertools_json(Message).Records[0]"
+S3_RAWSNS_SQS = "Records[*].powertools_json(body).Records[0]"
```

Comment:
you can use S3_SQS, because SNS Raw Message Delivery feature removes its wrapping. From a payload perspective, it's technically S3 -> SQS. If a customer now uses S3 -> SQS or S3 -> SNS [Raw Delivery] -> SQS - both works :) 

Same for any other RAW_SNS. This will improve UX since "Raw" might not be familiar to everyone unless you've used SNS for quite some time

Reply:
Thanks for the feedback @heitorlessa! I wasn't comfortable with those names (RAW) to be honest, your suggestion is better!